lib/inspec/runner_rspec.rb

```diff
@@ -90,9 +90,12 @@ def exit_code
       return @rspec_exit_code if @formatter.results.empty?
 
       stats = @formatter.results[:statistics][:controls]
+      load_failures = @formatter.results[:profiles]&.select { |p| p[:status] == "failed" }&.any?
       skipped = @formatter.results.dig(:profiles, 0, :status) == "skipped"
```

Comment:
I noticed this line _might_ be wrong, as it requires the first (`0`) profile to be skipped; when the run might have a passing profile ahead of the skipped profile.